packages/proxy/src/proxy.ts

```diff
@@ -243,18 +248,24 @@ export async function proxyV1({
     url = "/" + pieces.slice(2).map(encodeURIComponent).join("/");
   }
 
+  const isGoogleUrl = GOOGLE_URL_REGEX.test(url);
+
   const cacheableEndpoint =
     url === "/auto" ||
     url === "/embeddings" ||
     url === "/chat/completions" ||
     url === "/completions" ||
-    url === "/moderations";
+    url === "/moderations" ||
+    url === "/messages" ||
```

Comment:
should we prefix anthropic and google with `/v1/proxy/anthropic` etc to protect ourselves from providers creating overlapping names in the future?

Reply:
Done